<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Dell" name="com.qa.xoriant.training.ListUsers" tests="4" failures="0" timestamp="17 Sep 2020 11:36:30 GMT" time="7.353" errors="1">
  <testcase name="GetfirstNamesingRestAssuredBodyMethod" time="0.011" classname="com.qa.xoriant.training.ListUsers">
    <error type="org.testng.internal.reflect.MethodMatcherException" message="
Data provider mismatch
Method: GetfirstNamesingRestAssuredBodyMethod([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) pgno,(java.lang.String) 2]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
Data provider mismatch
Method: GetfirstNamesingRestAssuredBodyMethod([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) pgno,(java.lang.String) 2]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:45)
at org.testng.internal.Parameters.injectParameters(Parameters.java:796)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:983)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- GetfirstNamesingRestAssuredBodyMethod -->
  <testcase name="GetIDusingJsonPath" time="6.337" classname="com.qa.xoriant.training.ListUsers"/>
  <testcase name="UserListReqwithGiven" time="0.463" classname="com.qa.xoriant.training.ListUsers"/>
  <testcase name="UserListReq" time="0.542" classname="com.qa.xoriant.training.ListUsers"/>
</testsuite> <!-- com.qa.xoriant.training.ListUsers -->
